### PR TITLE
fix: support tooltipInfo on header cell (List & Table)

### PR DESCRIPTION
**Pull request by fabienhebert**
### Type

- Enhancement

### What has been done

Add a prop `tooltipInfo` on headerCell for both Table and List component.

This prop accepts a string and is optionel. If provided it add an info icon wrapped by a tooltip with the value as text

### Relevant logs and/or screenshots

Capture d’écran 2023-06-12 à 16 39 59

### CODE REVIEW

File: packages/ui/src/components/List/HeaderCell.tsx [hunk 1]
```diff
@@ -56,6 +57,7 @@ type HeaderCellProps = {
   isOrdered?: boolean
   orderDirection?: 'asc' | 'desc' | 'none'
   onOrder?: (newOrder: 'asc' | 'desc') => void
+  tooltipInfo?: string
```
Comment (matthprost) [inline packages/ui/src/components/List/HeaderCell.tsx:60]: Can we keep `tooltip` instead? That's the prop name we use for other components

File: packages/ui/src/components/Table/HeaderCell.tsx [hunk 2]
```diff
@@ -70,6 +71,7 @@ type HeaderCellProps = {
   width?: string
   minWidth?: string
   maxWidth?: string
+  tooltipInfo?: string
```
Comment (matthprost) [inline packages/ui/src/components/Table/HeaderCell.tsx:74]: Same here, `tooltip` as prop name is better

File: packages/ui/src/components/List/HeaderCell.tsx (hunk 1 above)
Comment (philibea) [reply packages/ui/src/components/List/HeaderCell.tsx:60]: info sound better IMHO

Comment (matthprost) [reply packages/ui/src/components/List/HeaderCell.tsx:60]: Let's go with `info` then 😄

**Review by JulienSaguez — APPROVED** [review]

**Review by matthprost — APPROVED** [review]

**Review by PetmePant — APPROVED** [review]

**Pull request merged**